src/entt/entity/storage.hpp

```diff
@@ -210,6 +274,16 @@ template<typename... Lhs, typename... Rhs>
     return !(lhs == rhs);
 }
 
+template<typename Traits, typename = void>
+struct is_component_trait_dynamically_sized
+    : std::false_type {};
+
+template<typename Traits>
```

Comment:
These are used only once. Here:
```cpp
static constexpr bool is_dynamically_sized_type_v = internal::is_component_trait_dynamically_sized<component_traits<Type>>::value;
```
Where the following would be enough anyway:
```cpp
static constexpr bool is_dynamically_sized_type_v = component_traits<Type>::has_dynamic_size;
```
To me, this class seems completely useless. Just yet another thing to maintain for no good reasons. Isn't it?

Reply:
I try to remove this specialization by moving everything into a specialized container.